LoopKitUI/Views/GlucoseRangePicker.swift

```diff
@@ -39,11 +39,26 @@ public struct GlucoseRangePicker: View {
     ) {
         self._lowerBound = Binding(
             get: { range.wrappedValue.lowerBound },
-            set: { range.wrappedValue = $0...range.wrappedValue.upperBound }
+            set: {
+                if $0 > range.wrappedValue.upperBound {
+                    // Prevent crash if picker gets into state where "lower bound" > "upper bound"
+                    range.wrappedValue = $0...$0
+                }
+                range.wrappedValue = $0...range.wrappedValue.upperBound
+                
+        }
         )
         self._upperBound = Binding(
             get: { range.wrappedValue.upperBound },
-            set: { range.wrappedValue = range.wrappedValue.lowerBound...$0 }
```

Comment:
[nit] seems generally useful.  Might it make sense to add a utility function like this?
```
func safeClosedRange(minimum: Int, maximum: Int) -> ClosedRange<Int> {
    if minimum > maximum { return minimum...minimum }
    return minimum...maximum
}
```
?

Reply:
Would need either two functions `closedRangeBoundedByMinimum` and `closedRangeBoundedByMaximum` or a method that takes an additional argument that specifies the type of resolution preferred (min or max).  Those don't sound worth it, unless this appears as a common pattern in the code.

Reply:
This is only the case in this one place, so I'm not convinced we need it (happy to revisit this later if we find more bugs like this)